specification/communication/data-plane/TrunkStatus/preview/2022-07-01-preview1/communicationservicestrunkstatus.json

```diff
@@ -0,0 +1,212 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Trunk health Service",
+    "description": "Trunk health Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2022-07-01-preview1"
+  },
+  "paths": {
+    "/sip/trunksHealth": {
+      "get": {
+        "tags": [
+          "TrunksHealth"
+        ],
+        "description": "This endpoint gets a list of all Trunks health configured for resource.",
+        "summary": "Gets a list of all Trunks health configured for resource.",
+        "operationId": "GetTrunksHealth",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/ListOfTrunksHealth"
```

Comment:
Avoid including type in the model name (`ListOf`).

You can follow chat's example and call this `TrunkHealthCollection`, or simply `TrunkHealths` (pluralized)

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/communication/data-plane/Chat/stable/2021-09-07/communicationserviceschat.json#L1238